coverage.clover

```diff
@@ -0,0 +1,1277 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think this file belongs in here ;)

Reply:
fixed 0d0ffea